app/src/main/java/org/piwigo/ui/settings/SettingsActivity.java

```diff
@@ -57,19 +58,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
         mPreferenceThumbnailSize = new ListPreference(getApplicationContext());
```

Comment:
Doesn't this create another instance of the preferences in addition to what was instantiated from the xml?

Reply:
@ramack Fixed.

Reply:
@ramack  Check the new commit